### ISSUE TITLE
Any Fluid into Mariculture tanks.

### ISSUE THREAD

**Issue by Xyrys** [joshiejack/Mariculture#282]
Hi,

Couldn't find this anywhere so thought I would submit it just in case.  Doesn't cause any issues I could see.  This log is from just one bucket of lava put into a mariculture tank.

http://pastebin.com/FX9DAqnm

Any liquid put from bucket to tank, or tank to bucket creates these reports.  Tried lava, water, creosote, oil, and fuel.

I am using FTB Regrowth pack for reference.

Only mods other than stock i have is Optifine and updated Buildcraft, Enderzoo, and Storage Drawers.

However could reproduce under the stock pack as well.

I don't see any messages with piped liquids though.

Xy.

**Comment by Xyrys** [comment]
An extra issue is the following which occurs when you place a tank with any liquid down as a block.

http://pastebin.com/gxXc1Xx1

**Comment by Xyrys** [comment]
Also found this in the forge log.  Related ??

http://pastebin.com/mC94dcTM

**Comment by joshiejack** [comment]
Did you update Mariculture as well? Regrowth has the non fixed fluids version.

**Comment by Xyrys** [comment]
I considered that before posting but since we are only using Forge 1291, rather than 1355+, I posted first before updating.

Updated to latest and just tested.  Tanks and buckets work fine now.. But the TConstruct error still occurs..  Will update Tconstruct and see if thats the issue.

**Comment by Xyrys** [comment]
Ok cannot update any further on TConstruct.. Its not a game breaking or stopping error, just thought I would post it incase you had not seen it.

Really enjoying Mariculture in regrowth..

**Comment by joshiejack** [comment]
If it was 1291 those errors wouldn't have been ocurring at all.

**Comment by Xyrys** [comment]
Definately 1291.  Just rechecked forge.

**Comment by Xyrys** [comment]
Just to be sure. 

Checking local assets file, for MC version1.7.10 Please wait! 
Setting up native libraries for Regrowth v Recommended Version MC 1.7.10
Syncing Assets:
Java Path: C:\Program Files\Java\jre1.8.0_40\bin\javaw.exe
Pack: Regrowth 1.7.10
Setting MinMemory to 256
Setting MaxMemory to 4096
Additional java parameters: -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:+DisableExplicitGC -XX:+AggressiveOpts 
Adding Optimization Arguments: -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:+CICompilerCountPerCPU -XX:+TieredCompilation
[19:46:03] [main/INFO] [LaunchWrapper]: Loading tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[19:46:03] [main/INFO] [LaunchWrapper]: Using primary tweak class name cpw.mods.fml.common.launcher.FMLTweaker
[19:46:03] [main/INFO] [LaunchWrapper]: Calling tweak class cpw.mods.fml.common.launcher.FMLTweaker
[19:46:03] [main/INFO] [FML]: Forge Mod Loader version 7.10.85.1291 for Minecraft 1.7.10 loading
[19:46:03] [main/INFO] [FML]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_40, running on Windows 7:amd64:6.1, installed at C:\Program Files\Java\jre1.8.0_40
[19:46:03] [main/INFO] [FML]: [AppEng] Core Init

**Comment by joshiejack** [comment]
Maybe on client, not neccessarily on server

**Comment by Xyrys** [comment]
Server as well.

GUI] The server is starting...
[GUI] Server started...
8:40:57 PM [main/INFO] [LaunchWrapper]: Loading tweak class name cpw.mods.fml.common.launcher.FMLServerTweaker
8:40:57 PM [main/INFO] [LaunchWrapper]: Using primary tweak class name cpw.mods.fml.common.launcher.FMLServerTweaker
8:40:57 PM [main/INFO] [LaunchWrapper]: Calling tweak class cpw.mods.fml.common.launcher.FMLServerTweaker
8:40:57 PM [main/INFO] [FML]: Forge Mod Loader version 7.10.85.1291 for Minecraft 1.7.10 loading
8:40:57 PM [main/INFO] [FML]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_40, running on Windows 7:amd64:6.1, installed at C:\Program Files\Java\jre1.8.0_40

**Comment by Xyrys** [comment]
You can't connect to mismatched forge versions can you ?  I can't say I have tried recently.. I know you couldn't way back when.

**Comment by joshiejack** [comment]
Yes you can.

**Comment by Xyrys** [comment]
Ok after further testing. It still happens with latest version using Forge 1291, but only in SMP.  When I originally posted that it was fixed I was in SP same world (copied direct from server) and the errors stopped.  Now that I have updated the server (even going so far as creating a new server with all files copied direct from client) the error persists.  Went back to SP.  Error gone..  Don't know what to add now.  Have triple checked forge version on both server and client.. Both are definitely 1291. All configs and mods the same.

Its a client side error too.. I cannot get it to appear in any logs on server when it occurs.

Here is error log for latest version.

http://pastebin.com/2m56YCFh
